.clang-format

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Could you make this match as much as possible with [SDK clang-format](https://github.com/keystone-enclave/keystone-sdk/blob/master/.clang-format)?

Reply:
Yes for sure, I didn't notice that the SDK also had a clang-format